ai/writing-assistant.rst

```diff
@@ -0,0 +1,83 @@
+Writing Assistant
+=================
+
+Zammad's writing assistant tools allow agents to apply AI based changes to
+their text before saving or sending it. You can access the settings in Zammad's
+admin section under *AI > Writing Assistant*. Make sure to configure an
+:doc:`AI provider <provider>` before activating the feature. Otherwise, a
+warning message will tell you to do so.
+
+Default Tools
+-------------
+
+Zammad ships some default tools:
+
+- Improve writing
+- Fix spelling and grammar
+- Expand
+- Simplify
+
+These are useful basic tools for all agents. This is why they are not
+customizable and deletable. If you enable the feature by turning the
+**Writing Assistant** toggle on, these default tools are available to all
+agents.
+
+Note that these built-in tools can benefit from optimizations which can be
+shipped by an update of Zammad.
+
+Add Custom Tools
+----------------
+
+You can add your own custom write tools by providing instructions for the AI
```

Comment:
`write tools` => `writing assistant tools`

Let's try to be consistent with the terminology.